### PR TITLE
Add niu mini dye sub keymap

### PR DESCRIPTION
**Pull request by chusjp**
## Description

This PR is for adding a layout that I was missing in this repo and that has been really useful to me and I think it would be for people having the same keycaps set.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/niu_mini/keymaps/dyesub/keymap.c [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+    }
+
+    if (IS_LED_ON(usb_led, USB_LED_KANA)) {
+
+    } else {
+
+    }
+
+}
```
Comment (zvecr) [inline keyboards/niu_mini/keymaps/dyesub/keymap.c:103]:
```suggestion
```

File: keyboards/niu_mini/keymaps/dyesub/rules.mk [hunk 2]
```diff
@@ -0,0 +1,4 @@
+MOUSEKEY_ENABLE = no
+BACKLIGHT_ENABLE = no
+AUDIO_ENABLE = no
+CONSOLE_ENABLE = no         # Console for debug(+400)
```
Comment (zvecr) [inline keyboards/niu_mini/keymaps/dyesub/rules.mk:4]:
```suggestion
CONSOLE_ENABLE = no
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by Erovia — APPROVED** [review]

File: keyboards/niu_mini/keymaps/dyesub/keymap.c [hunk 3]
```diff
@@ -0,0 +1,58 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/niu_mini/keymaps/dyesub/keymap.c:1]: Could you add a GPL2+ compatible license header at the beginning of the file, here?

Comment (chusjp) [reply keyboards/niu_mini/keymaps/dyesub/keymap.c:17]: Sure, it's added now.

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**